src/factions/slaves_to_darkness/units.ts

```diff
@@ -1034,6 +1034,16 @@ const Units = {
       },
     ],
   },
+  'Khagra`s Ravagers': {
```

Comment:
```suggestion
  "Khagra's Ravagers": {
```
Single quote for the apostrophe instead of a backtick; double quotes to allow that.